charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-apiserver.rules.yaml

```diff
@@ -71,3 +71,15 @@ groups:
     annotations:
       description: 'The API server ({{ $labels.instance }}) is using {{ $value }}% of the available file/socket descriptors.'
       summary: 'The API server has too many open file descriptors'
+  ### API auditlog ###
+  - alert: KubeApiServerTooManyAuditlogFailures
```

Comment:
Please add a test for this in `charts/seed-monitoring/charts/core/charts/prometheus/rules-tests/kube-apiserver.rules.test.yaml`